requirements.txt

```diff
@@ -18,15 +18,14 @@ tqdm>=4.33.0,<5.0.0
 
 # Internal models
 scikit-learn>=0.20.2,<0.22.0
-torch>=1.1.0,<1.2.0
+torch>=1.4.0,<2.0.0
```

Comment:
Is this incompatible with 1.2 and 1.3?

Reply:
yes, all tests pass with the older versions 1.2.0 and 1.3.0.
relaxing the requirement to `torch>=1.2.0,<2.0.0`